form_data.js

```diff
@@ -28,3 +28,29 @@ for (let i = 0; i < inputs.length; i++) {
     }
```

Comment:
Please use linter config for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) config.

Reply:
thanks but this is the file I am using. I don't know if you realize the kind of errors that are raised (no more), however it is a common and recommended syntax

Reply:
Hi, It's allowed to use linter configs which are developed by the Microverse team for students. So please use the suggested code linter.